currency/withdraw.cc.lua

```diff
@@ -0,0 +1,67 @@
+{{/*
+		Simple Currency System By GasInfinity
+        
+        Recommended Trigger: Regex: "^-(withdraw|with)"
+        
+        Withdraw Your Money from The Bank, Usage "-withdraw <Amount> or all " 
+*/}}
+
+{{/* CONFIGURATION START*/}}
+{{ $currency := "💰" }}
+{{$bankName := "GBank"}}
+{{$minAmountToWithdraw := 1}}
+{{/*CONFIGURATION END*/}}
+
+{{/*MAIN CODE!!*/}}
+
+{{ $dbHandName := "HAND" }} 
+{{ $dbBankName := "GBANK" }}
+
+{{$error1 := "**Error, Syntax is !withdraw <all> or <amount>**"}}
+{{$error2 := "**Error, You Need At Least: **"}}
+{{$withall := "all"}}

```

Comment:
Extract to configuration variable, consider more descriptive error names.

Reply:
done, i did the same as with -deposit